pinot-common/src/main/java/org/apache/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -130,7 +130,13 @@ public static String constructPropertyStorePathForSegmentLineage(String tableNam
     return StringUtil.join("/", PROPERTYSTORE_SEGMENT_LINEAGE, tableNameWithType);
   }
 
-  public static String constructPropertyStorePathForMinionTaskMetadata(String taskType, String tableNameWithType) {
+  public static String constructPropertyStorePathForMinionTaskMetadataNewFormat(
```

Comment:
Suggest making this `constructPropertyStorePathForMinionTaskMetadata` and changing the current one to `constructPropertyStorePathForMinionTaskMetadataDeprecated` and annotate it as deprecated

Reply:
Please check the code style setting. Seems it is not using the Pinot Style

Reply:
interesting, the format does not take effects until I manually reformat the file 🤔